src/api/adamik/broadcast.ts

```diff
@@ -8,6 +8,13 @@ export type BroadcastResponse = {
   error?: { message: string };
 };
 
+export type BackendErrorResponse = {
+  status: {
+    errors: Array<{ message: string }>;
+    warnings: Array<string>;
```

Comment:
Technically both have the same format:
```suggestion
    errors: Array<{ message: string }>;
    warnings: Array<{ message: string }>;
```